StyleCop.Analyzers/StyleCop.Analyzers/StyleCop.Analyzers.csproj

```diff
@@ -20,6 +20,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="1.2.1" />
     <PackageReference Include="Nullable" Version="1.3.1" PrivateAssets="all" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
```

Comment:
This is a change I would not accept at this point. Sam has rejected this in the original repository before because of concerns that this might not work in really old Roslyn/VS versions. If Sam does not return and this repository survives, I plan on removing support for older versions gradually and then this would eventually be fine, but I am not ready to do this right now.